index.js

```diff
@@ -23,8 +23,8 @@ async function parse ({path: modulePath, files, format, importMap, output}) {
 
 	log.info(`Parsing ${modulePath} ...`);
 	documentation.build(files, {shallow: true}).then(
-		(root) => {
-			let result = encodeModule({root, section: root, parent: root, importMap, log}).join('\n');
+		async (root) => {
```

Comment:
Too many async/await added...
We can't avoid the change to async/await, but is there any other way to minimize changes?

Reply:
We didn't find any other ways to avoid these changes as long as these functions are interconnected but if anyone from SWP has another solution don't hesitate to suggest it.

Reply:
I couldn't find a good way either.
These changes will be merged.
Please update the code if you find a good way later.